src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1286,21 +1286,142 @@ export default class DataTable extends Vue {
     }
   }
 
+  _printHead() {
+    return (
+      <thead>
+        <tr>
+          {Object.keys(this.data.head).map((column: string) => {
+            return <th>{this.data.head[column].label}</th>;
+          })}
+        </tr>
+      </thead>
+    );
+  }
+
+  _printFooter() {
+    const resultsCount =
+      this.mode === DataTableModes.CLIENT
+        ? this._showSelectedOnly
+          ? this._getSelectedFirstLevelRowsCount()
+          : this.data.body.length
+        : this.config.pagination.results;
+
+    return <div class="-p--2">{resultsCount} results</div>;
```

Comment:
I suggest you give this element a specific class, like `chi-data-table__print-footer`, and apply the padding to it.